README.md

```
@@ -59,16 +59,16 @@ tokmd diff main HEAD
 ### LLM Context Packing
 Pack files into an LLM context window with budget-aware selection:
 ```bash
-tokmd context --budget 128k --output bundle > context.txt  # Ready to paste
-tokmd context --budget 200k --strategy spread              # Coverage across modules
-tokmd context --budget 100k --output bundle --compress     # Stripped for density
+tokmd context --budget 128k --output bundle --out context.txt  # Ready to paste
+tokmd context --budget 200k --strategy spread                  # Coverage across modules
+tokmd context --budget 100k --output bundle --compress --out context.txt  # Strip blank lines for density
 ```
 
 ### LLM Context Planning
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)